web/pandas/getting_started.md

```diff
@@ -22,11 +22,28 @@ by [Wes McKinney](https://wesmckinney.com/), creator of pandas.
 
 ## Videos
 
-<iframe width="560" height="315" frameborder="0"
-src="https://www.youtube.com/embed/_T8LGqJtuGc"
-allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-allowfullscreen></iframe>
+<iframe
+  src="https://www.youtube.com/embed/_T8LGqJtuGc"
+  style="width: 100%; max-width: 560px; height: 315px;"
+  frameborder="0"
+  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
 
 ## Cheat sheet
 
 [pandas cheat sheet](https://pandas.pydata.org/Pandas_Cheat_Sheet.pdf)
+
+## Try pandas in your browser (experimental)
+
+You can try pandas in your browser with the following interactive shell
+without needing to install anything on your system.
+
+**Please note it can take a while (>30 seconds) before the shell is initialised and ready to run commands.**
```

Comment:
For consistency with the rest of the docs maybe?

```suggestion
**Please note it can take a while (>30 seconds) before the shell is initialized and ready to run commands.**
```